charts.go

```diff
@@ -41,9 +41,22 @@ func ScenarioLineChart(scen *Scenario) template.HTML {
 		}
 	}
 
+	wmaDates := []string{}
+	wmaScores := []opts.LineData{}
+	for _, dateScore := range scen.ByDateWMA {
+		for date, dateWMA := range dateScore {
+			wmaDates = append(wmaDates, SimplifyDate(date))
+			wmaScores = append(wmaScores, opts.LineData{
+				Name:  fmt.Sprintf("%v: %v. Grouped: %v", SimplifyDate(date), dateWMA.Avg, dateWMA.Grouped),
+				Value: dateWMA.Avg,
+			})
+		}
+	}
+
 	line.SetXAxis(maxDates).
 		AddSeries("Max scores", maxScores).
 		AddSeries("Average scores", avgScores).
+		AddSeries(strconv.Itoa(DefaultWMAWindow)+"-day Moving Average", wmaScores).
```

Comment:
![image](https://user-images.githubusercontent.com/20703677/129945776-ecb96a3e-c648-449d-87a2-67dc6fa98951.png)

Casing should match to be consistent with the rest:
`-day moving average`